deploy/docker/entrypoint.sh

```diff
@@ -168,7 +168,8 @@ init_replica_set() {
     # Check mongodb cloud Replica Set
     echo "Checking Replica Set of external MongoDB"
 
-    if [[ $(mongo "$APPSMITH_MONGODB_URI" --quiet --eval "rs.status().ok") -eq 1 ]]; then
+    mongo_state=$(mongo --host $APPSMITH_MONGODB_URI --quiet --eval "rs.status().ok")
```

Comment:
Can we quote all variable references please? 🙂

```suggestion
    mongo_state="$(mongo --host "$APPSMITH_MONGODB_URI" --quiet --eval "rs.status().ok")"
```